TechItEasyDatabase_GW.sql

```diff
@@ -0,0 +1,125 @@
+DROP TABLE IF EXISTS Users;
+DROP TABLE IF EXISTS TelevisionCIModules;
+DROP TABLE IF EXISTS TelevisionWallBrackets;
+DROP TABLE IF EXISTS Televisions;
+DROP TABLE IF EXISTS RemoteControllers;
+DROP TABLE IF EXISTS CIModules;
+DROP TABLE IF EXISTS WallBrackets;
+
+
+CREATE TABLE Users
+(
+	username VARCHAR(50) NOT NULL UNIQUE,
+	password VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE RemoteControllers
+(
+-- Attributen vanuit de abstracte Product klasse
+	productid INT PRIMARY KEY,
```

Comment:
ik mis wel in deze tabel  iets dat ervoor zorgt dat de primary key altijd uniek is , zoals serial of unique. Verderop bij de koppeltabel en de wallbracket tabel doe je dat wel.

Reply:
De PRIMARY KEY zorgt ervoor dat de productid uniek moet zijn en niet NULL mag zijn.
Ik wist het niet meer precies en heb het even terug gezocht, op deze pagina vindt je handig overzicht:
https://www.w3schools.com/sql/sql_constraints.asp